src/index.html

```diff
@@ -0,0 +1,12 @@
+<html>
+
+<head>
+</head>
```

Comment:
Please add the following code `<meta name="viewport" content="width=device-width, initial-scale=1" />` in the `head` of the `index.html` as it will enables us to control the width and scaling of the viewport. Oherwise, mobile devices render pages at typical desktop screen widths, and then scale the pages to fit the mobile screens and that will affect our UI.